### PR TITLE
VisualConcept unit tests

### PR DESCRIPTION
**Pull request by antares1470**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Added some unit tests for `VisualConcept`. Since there were no regex's or similar to test for attributes in the concept, the only tests included are just to test whether attributes have been added correctly.

### Why Should This Be In Core?

more unit tests = good

### Benefits

more unit tests

### Applicable Issues

#413 #337

### CODE REVIEW

File: CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConceptNGTest.java
Comment (arcturus2) [inline CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConceptNGTest.java:2]: Just noticed the year could be updated when you get a chance? I missed this on your other PR's so happy for this to be done from new PR's given we need to do search replace at some point anyway.

**Review by arcturus2 — APPROVED** [review]
Nice work @antares1470

File: CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConceptNGTest.java
Comment (antares1470) [reply CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConceptNGTest.java:2]: Good catch. I've updated it for future PRs but yeah, given a PR is likely to be created at some stage to update it across the codebase at some stage, I'll hold off provided it's not created before this PR gets merged in

**Review by aldebaran30701 — APPROVED** [review]

**Pull request merged**
